liquibase-core/src/main/java/liquibase/integration/spring/SpringResource.java

```diff
@@ -59,10 +60,13 @@ public InputStream openInputStream() throws IOException {
     }
 
     @Override
-    public OutputStream openOutputStream(boolean createIfNeeded) throws IOException {
-        if (!resource.exists() && !createIfNeeded) {
+    public OutputStream openOutputStream(OpenOptions openOptions) throws IOException {
+        if (!resource.exists() && !openOptions.isCreateIfNeeded()) {
```

Comment:
This needs a null check on openOptions

Reply:
This is identified as not null in the javadocs, and adding null checks everywhere it is used makes the code pretty complicated, so I'm not going to do this.